pkg/app/handlers/config.go

```diff
@@ -0,0 +1,62 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+)
+
+type ConsoleCfg struct {
+	AsertoDirectoryURL       string  `json:"asertoDirectoryUrl"`
+	AuthorizerServiceURL     string  `json:"authorizerServiceUrl"`
+	AuthorizerAPIKey         string  `json:"authorizerApiKey"`
+	DirectoryAPIKey          string  `json:"directoryApiKey"`
+	DirectoryTenantID        string  `json:"directoryTenantId"`
+	AsertoDirectoryReaderURL *string `json:"asertoDirectoryReaderUrl,omitempty"`
+	AsertoDirectoryWriterURL *string `json:"asertoDirectoryWriterUrl,omitempty"`
+	AsertoDirectoryModelURL  *string `json:"asertoDirectoryModelUrl,omitempty"`
```

Comment:
@BogdanIrimie could we create some symmetry between authoriser and directory?

Let me know if this looks good

```
AuthorizerServiceURL     string  `json:"authorizerServiceUrl"`
AuthorizerAPIKey         string  `json:"authorizerApiKey"`
DirectoryServiceURL       string  `json:"DirectoryServiceUrl"`
DirectoryAPIKey          string  `json:"directoryApiKey"`
DirectoryTenantID        string  `json:"directoryTenantId"`
DirectoryReaderServiceURL *string `json:"DirectoryReaderServiceUrl,omitempty"`
DirectoryWriterServiceURL *string `json:"DirectoryWriterServiceUrl,omitempty"`
DirectoryModelServiceURL  *string `json:"DirectoryModelServiceUrl,omitempty"`
```